### PR TITLE
Task#5280 handle event pause next previous

### PR DESCRIPTION
**Pull request by NguyenQuocBao19182**
Link redmine : https://edu-redmine.framgia.vn/issues/5280

### CODE REVIEW

**Comment by NguyenQuocBao19182** [comment]
ready

File: app/src/main/java/com/framgia/music_49/screen/listmusic/ListMusicFragment.java [hunk 1]
```diff
@@ -51,6 +57,8 @@ private void initPresenter() {
     }
 
     private void initView(View view) {
+        mSongs = new ArrayList<>();
+        mNavigator=new Navigator();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_49/screen/listmusic/ListMusicFragment.java]: reformat code

File: app/src/main/java/com/framgia/music_49/screen/playsong/PlaySongFragment.java [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        initData();
+        intView(view);
+        startBoundService();
+        return view;
+    }
+
+    private void initData() {
+        mSongs = new ArrayList<>();
+        Bundle bundle = getArguments();
+        if (bundle != null) {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_49/screen/playsong/PlaySongFragment.java]:
```suggestion
        if (getArguments() != null) {
```

**Comment by NguyenQuocBao19182** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**